### PR TITLE
Handle non valid params in queries for kraken staking

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

File: rotkehlchen/api/rest.py [hunk 1]
```diff
@@ -3947,9 +3947,9 @@ def _query_kraken_staking_events(
 
         if exchanges_list is None:
             return {
```
Comment (LefterisJP) [inline rotkehlchen/api/rest.py:3949]: Both for this and below we have a function to do this dict wrapping: `wrap_in_fail_result()`

File: rotkehlchen/api/v1/encoding.py [hunk 2]
```diff
@@ -1117,6 +1117,8 @@ def make_staking_query(  # pylint: disable=no-self-use
             **_kwargs: Any,
     ) -> Dict[str, Any]:
         order_by_attribute = data['order_by_attribute'] if data['order_by_attribute'] is not None else 'timestamp'  # noqa: E501
+        if order_by_attribute == 'event_type':
```
Comment (LefterisJP) [inline rotkehlchen/api/v1/encoding.py:1120]: Ah wait I misunderstood when we spoke in discord. What I meant is that they give you a "type" argument and you map it to whatever you need. If they give you something they should not give you it's a BAD REQUEST error and should fail the call

Comment (yabirgb) [reply rotkehlchen/api/v1/encoding.py:1120]: Yes the code behaves in that way. I remap the only possible different value (this one) and any other invalid param is treated as a bad request

Comment (LefterisJP) [reply rotkehlchen/api/v1/encoding.py:1120]: But that's not what I meant. The interface between frontend and backend is the api and it's a contract that should be respected.

If they give an argument they should not, then that should be an error and not quietly mapped to something else.

**Review by LefterisJP — APPROVED** [review]
LGTM after clarifications in the chat

**Pull request merged**